cfme/fixtures/embedded_ansible.py

```diff
@@ -0,0 +1,81 @@
+import pytest
+from wrapanapi.utils.random import random_name
+
+
+def create_namespaces(d, namespace_names):
+    namespace = None
+    for namespace_name in namespace_names:
+        if namespace:
+            namespace = namespace.namespaces.instantiate(namespace_name)
+        else:
+            namespace = d.namespaces.instantiate(namespace_name)
+    return namespace
+
+
+def copy_ae_instance(appliance, source, destination_domain_name):
+    """
+    Copies source AE instance to destination domain and returns copied AE instance
+    """
+    source = source.split('/')
+    source_domain_name = source.pop(0)
+    source_instance_name = source.pop(-1)
+    source_class_name = source.pop(-1)
+    source_domain = appliance.collections.domains.instantiate(source_domain_name)
+    namespace = create_namespaces(source_domain, source)
+    namespace.classes.instantiate(source_class_name)\
+        .instances.instantiate(source_instance_name).copy_to(destination_domain_name)
+
+    destination_domain = appliance.collections.domains.instantiate(destination_domain_name)
+    destination_namespace = create_namespaces(destination_domain, source)
+    return destination_namespace.classes.instantiate(source_class_name)\
+        .instances.instantiate(source_instance_name)
+
+
+@pytest.fixture
+def domain(request, appliance):
+    """
+    Created new domain with random name, if we want we can also provide a custom name with
+    @pytest.mark.parametrize test decorator.
+
+    Example: @pytest.mark.parametrize('domain', ['TestDomain'], indirect=True)
+    """
+    dc = appliance.collections.domains
+    d = dc.create(
+        name=getattr(request, 'param', None) or random_name(),
+        enabled=True)
+    yield d
+    d.delete()
+
+
+@pytest.fixture
+def copy_ae_instance_to_domain_name(request, appliance):
```

Comment:
Since we don't make use of this fixture ATM I'd just remove it. It's simple enough to implement it later if needed.